stages/org.osbuild.coreos.live-artifacts.mono

```diff
@@ -923,7 +926,9 @@ def main(workdir, tree, inputs, options, loop_client):
 
     mk_osmet_files(deployed_tree, img_metal, img_metal4k, loop_client, paths, os_release)
 
-    blsentry_kargs = mkrootfs_metal(paths, workdir, img_metal, fstype, fsoptions, loop_client)
+    (blsentry_kargs, osname) = mkrootfs_metal(paths, workdir, img_metal, fstype, fsoptions, loop_client)
+    if not osname:
+        osname = f"{os_release['ID']}-{os_release['VARIANT_ID']}"
```

Comment:
Super minor since I don't know if it applies in your case but:

Since `VARIANT_ID` can be empty (or not exist, I guess?) would you want to use `'-'.join([os_release['ID'], os_release['VARIANT_ID']])` here so there's no dangling `-`?